common/djangoapps/xmodule_modifiers.py

```diff
@@ -277,6 +277,7 @@ def add_staff_markup(user, has_instructor_access, block, view, frag, context):
                      'category': str(block.__class__.__name__),
                      # Template uses element_id in js function names, so can't allow dashes
                      'element_id': block.location.html_id().replace('-', '_'),
+                     'region': unicode(course_id).replace(':', '').replace('/', '_').replace('+', '_'),
```

Comment:
It's a bit unfortunate that this replacement is done twice (once here, and one in the `sudo_required` decorator). Could this simply pull the active region? Or could you extract the function that maps from `CourseKey` to region?